content/nomos/updates/2023-11-13.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 2023-11-06 Nomos weekly
+tags:
+  - nomos-updates
+date: 2023-11-13
+lastmod: 2023-11-13
+draft: false
+description: Weekly update of Nomos
+---
+
+## `network privacy and mixnet`
+
+### `research`
+- Analysis of failure probability in random partitions of networks constructed from nodes with prescribed voting “weights”:  
+- Considered a scenario when weights and Byzantine labels of nodes are sampled once, i.e. fixed, and then assigned to committees randomly.
+- A derivation of a probability distribution for the random process is currently in progress
+- Considered designing a gradient descent algorithm which, given weights and Byzantine labels of nodes, tries to find assignments to committees with the smallest number of failures. 
+- Analysis and implementation of the above algorithm is currently in progress. -reviewed literature on analysis of Loopix mixnets and implemented code for a simulation which computes fraction of de-anonymized messages.  
+- The details on all of the above are provided in [https://www.overleaf.com/read/kkmsngmcgbkj#c4de95](https://www.overleaf.com/read/kkmsngmcgbkj#c4de95 "https://www.overleaf.com/read/kkmsngmcgbkj#c4de95")
+
+### `development`
+
+- Walked through core parts of Nym implementation again to get details of what are written in papers.
+- Clarified sphinx packet creation process (+ encryption): https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2#d0bbc9d1f63e43faaa30bb4c888102bd]
+- Clarified delay calculation in Poisson distribution (inspired to Nym impl): [https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#3ae7e03fcbad461ab8d6b57e5c0e88fe](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#3ae7e03fcbad461ab8d6b57e5c0e88fe "https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#3ae7e03fcbad461ab8d6b57e5c0e88fe")
+- Clarified loop cover traffic creation & interval in Poisson (inspired to Nym impl): [https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#14f53c5d6c844c828689f0412d5e2195](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#14f53c5d6c844c828689f0412d5e2195 "https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#14f53c5d6c844c828689f0412d5e2195")
+- Suggesting skipping two other types of cover traffic (at least, for now)
+	- Drop cover by nomos-node: https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#76993c1312ea464a88557987a2f37b60
+		- Loop cover by mix-node: https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?pvs=4#f07a473a4b5d4f338ff024a145f6b525
```

Comment:
All these documents in Notion are actually specifications, so they fall onto research